### DISCUSSION TITLE
"Horizontal" menu disappeared?

### DISCUSSION THREAD

**Discussion by gabopushups** [OpenHantek/OpenHantek6022#209] [Q&A] [answered]
Just until last night, the program was running fine. But today, my horizontal menu disappeared completely. I can't find it anywhere, and by right clicking, the options that appear are greyed out, and the "horizontal" is unticked and I can't click it.

Any ideas?

![image](https://user-images.githubusercontent.com/65993380/129971775-6d7a76e6-f7bd-4a8c-9d86-e7ac0519bdcd.png)

**Comment by gabopushups** [comment]
I found a workaround that required modifying the code.

I went to openhantek/src/docks/ and then modified the file "HorizontalDock.cpp" by adding the function `show()` under the loadSettings function.

```
void HorizontalDock::loadSettings( DsoSettingsScope *scope ) {
    show(); // Show the missing dock
    // Set values
    setSamplerate( scope->horizontal.samplerate );
    setTimebase( scope->horizontal.timebase );
    setFormat( scope->horizontal.format );
    setCalfreq( scope->horizontal.calfreq );
}
```

And then had of course to rebuild the program. It's not the most elegant solution, but it worked for me.

**Comment by Ho-Ro** [comment, marked as answer]
Please see the correct solution: #210
